Cython/Utility/CythonFunction.c

```diff
@@ -1358,6 +1345,20 @@ __pyx_FusedFunction_call(PyObject *func, PyObject *args, PyObject *kw)
     return result;
 }
 
+static PyObject *
+__Pyx_FusedFunction_get_self(__pyx_FusedFunctionObject *m, CYTHON_UNUSED void *closure)
+{
+    PyObject *self;
+
+    self = m->self;
+    if (self == NULL) {
```

Comment:
Let's consider the exception case "unlikely", simply because it's so costly any way.
```suggestion
    PyObject *self = m->self;
    if (unlikely(!self)) {
```